packages/privacy-toolset/src/cookie-banner/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState } from 'react';
+import { CustomizedConsent } from './customized-consent';
+import { SimpleConsent } from './simple-consent';
+import { Buckets, CustomizedConsentContent, SimpleConsentContent } from './types';
+
+import './styles.scss';
+
+export type CookieBannerProps = {
+	content: {
+		simpleConsent: SimpleConsentContent;
+		customizedConsent: CustomizedConsentContent;
+	};
+	onAccept: ( buckets: Buckets ) => void;
+};
+
+enum ConsentViewType {
+	Simple,
+	Customized,
+}
+
+export const CookieBanner = ( { content, onAccept }: CookieBannerProps ) => {
+	const [ viewType, setViewType ] = useState< ConsentViewType >( ConsentViewType.Simple );
+	const handleSetCustomizedView = () => {
+		setViewType( ConsentViewType.Customized );
+	};
```

Comment:
I am not sure this is a huge deal, but I think it would be nice to have an ability to switch back to the Simple view instead of locking it in the Customized view

Reply:
I was thinking about this, but I think once we have the user choose cutomize (to set custom cookie preferences), they still have the option to choose all cookies, so I don't think there's a need to "go back" to the collapsed cookie banner. We could add it though, as a follow-up, it's an easy change. :)

Reply:
We leave it as-is in this PR, and we can always come back to it in the future. From the perspective of UX, I don't think we need to create a "go back" button, as everything is still achievable from the "customized" view.